backend/tracim_backend/lib/core/system.py

```diff
@@ -23,11 +24,20 @@ def get_about(self) -> AboutModel:
         )
 
     def get_config(self) -> ConfigModel:
+        collaborative_document_edition_config = None
+        if self._config.COLLABORATIVE_DOCUMENT_EDITION__ACTIVATED:
+            collaborative_document_edition_api = CollaborativeDocumentEditionFactory().get_collaborative_document_edition_lib(
+                session=None, current_user=None, config=self._config
```

Comment:
`session` is not typed as Optional. You should change it's type.

Reply:
done